.eslintrc

```diff
@@ -0,0 +1,37 @@
+{
+  "extends": [
+    "amo",
+    "plugin:amo/recommended",
+    "plugin:jest/recommended",
+    "plugin:@typescript-eslint/recommended"
```

Comment:
I see that they clobbered some of our AMO preferences. I discovered that `no-used-vars` is a warning but it should be an error. Maybe there are more.

```diff
diff --git a/.eslintrc b/.eslintrc
index cbae414..f70c925 100644
--- a/.eslintrc
+++ b/.eslintrc
@@ -25,7 +25,8 @@
     ],
     // We decided to prefer types over interfaces.
     "@typescript-eslint/prefer-interface": "off",
-    "@typescript-eslint/explicit-member-accessibility": "off"
+    "@typescript-eslint/explicit-member-accessibility": "off",
+    "@typescript-eslint/no-unused-vars": "error"
   },
   "settings": {
     "import/resolver": {
```

Reply:
Agreed!

My problem currently is that this rule reports false-positive when vars are used in JSX.